crates/uv-cli/src/lib.rs

```diff
@@ -2452,18 +2427,13 @@ pub struct TreeArgs {
     #[arg(long, conflicts_with = "universal")]
     pub python_platform: Option<TargetTriple>,
 
-    /// The Python interpreter for which packages should be listed.
+    /// The Python interpreter to use for resolution.
     ///
-    /// By default, uv installs into the virtual environment in the current working directory or
-    /// any parent directory. The `--python` option allows you to specify a different interpreter,
-    /// which is intended for use in continuous integration (CI) environments or other automated
-    /// workflows.
+    /// A Python interpreter is required to perform the lock before displaying
+    /// the tree.
     ///
-    /// Supported formats:
-    /// - `3.10` looks for an installed Python 3.10 using `py --list-paths` on Windows, or
-    ///   `python3.10` on Linux and macOS.
-    /// - `python3.10` or `python.exe` looks for a binary with the given name in `PATH`.
-    /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
+    /// See `uv help python` for details on Python discovery and supported
+    /// request formats.
```

Comment:
This is not correct, moved this fix into https://github.com/astral-sh/uv/pull/5870